lib/hawkular/alerts/alerts_api.rb

```diff
@@ -348,6 +348,8 @@ def initialize(trigger_hash)
       @context = trigger_hash['context']
       @type = trigger_hash['type']
       @tags = trigger_hash['tags']
+      @firing_match = trigger_hash['firingMatch']
+      @auto_resolve_match = trigger_hash['autoResolveMatch']
```

Comment:
camel case is not very "Rubyish" (unlike Java), so unless these 2 parameters are coming from somewhere else that I missed, it is better to have lowercase and "_" kind of format.